src/components/CreateList/index.js

```diff
@@ -0,0 +1,22 @@
+import { getToken } from '@the-collab-lab/shopping-list-utils';
+import { useLocalStorage } from '../../hooks/useLocalStorage';
+import { useHistory } from 'react-router-dom';
+
+export const CreateList = () => {
+  let history = useHistory();
+  const push = history.push;
```

Comment:
We could have something like this using destructuring:
```suggestion
  const { push } = useHistory();
```

Reply:
I sent this too with the last commit! thanks for the destructuring tip! :laughing: